### DISCUSSION TITLE
How-to invoke another-quickswitcher?

### DISCUSSION THREAD

**Discussion by craziedde** [tadashi-aikawa/obsidian-another-quick-switcher#265] [General]
1. What is the default hotkey to invoke this plugin and then switch to custom tag search?
2. Is it possible to combine file-name-search + tag-search + header-search (for example)?

Thank you
@craziedde

**Comment by tadashi-aikawa** [comment]
@craziedde 
Sorry for my late answer.

> What is the default hotkey to invoke this plugin and then switch to custom tag search?

The 'custom tag search' command is not provided by default.

> Is it possible to combine file-name-search + tag-search + header-search (for example)?

If you want the search query to target file names, tag names, and header names, please use the following configured command:

- **Search by:** Enable `Tag` and `Header`
- **Search target:** `file` (default)

![image](https://github.com/user-attachments/assets/d4f04b6a-8fde-49ac-b262-79574d2bf799)

**Reply by craziedde** [reply]
Hi @tadashi-aikawa 

> If you want the search query to target file names, tag names, and header names, please use the following configured command:
> 
> * **Search by:** Enable `Tag` and `Header`
> * **Search target:** `file` (default)

        Thank you for reply.
        Is there a good video tutorial on how-to use another-quickswitcher?
        I'm a little confused on general usage of the plugin

**Reply by tadashi-aikawa** [reply]
@craziedde 
> Is there a good video tutorial on how-to use another-quickswitcher?

Currently, there doesn't seem to be an official video tutorial available for Another Quick Switcher.

> I'm a little confused on general usage of the plugin

I have extracted the key points that are emphasized the most from the beginning section of the README.

- Creating custom search commands (`Custom searches`)
- Customizing the hotkeys in the quick switcher to your preference
- Searching backlinks and moving them **without leaving the keyboard** (`Backlink search`)
- Searching **regardless of the order in which tokens appear**
- Not performing very fuzzy searches by default, but an option is available to enable them
